docs/gallery/plot_dynamictable_tutorial.py

```diff
@@ -5,31 +5,30 @@
 DynamicTable Tutorial
 =====================
 
-This is a tutorial for interacting with ``DynamicTable`` objects. This tutorial
+This is a tutorial for interacting with :py:class:`~hdmf.common.table.DynamicTable` objects. This tutorial
 is written for beginners and does not describe the full capabilities and nuances
-of ``DynamicTable`` functionality. Please see the :ref:`dynamictable-howtoguide`
+of :py:class:`~hdmf.common.table.DynamicTable` functionality. Please see the :ref:`dynamictable-howtoguide`
 for more complete documentation. This tutorial is designed to give
-you basic familiarity with how ``DynamicTable`` works and help you get started
-with creating a ``DynamicTable``, adding columns and rows to a ``DynamicTable``,
-and accessing data in a ``DynamicTable``.
+you basic familiarity with how :py:class:`~hdmf.common.table.DynamicTable` works and help you get started
+with creating a :py:class:`~hdmf.common.table.DynamicTable`, adding columns and rows to a
+:py:class:`~hdmf.common.table.DynamicTable`,
+and accessing data in a :py:class:`~hdmf.common.table.DynamicTable`.
 
-"""
+Introduction
+------------
+The :py:class:`~hdmf.common.table.DynamicTable` class represents a column-based table
+to which you can add custom columns. It consists of a name, a description, a list of
+row IDs, and a list of columns.
 
-###############################################################################
-# Introduction
-# ------------
-# The :py:class:`~hdmf.common.table.DynamicTable` class represents a column-based table
-# to which you can add custom columns. It consists of a name, a description, a list of
-# row IDs, and a list of columns.
+Constructing a table
+--------------------
+To create a :py:class:`~hdmf.common.table.DynamicTable`, call the constructor for
+:py:class:`~hdmf.common.table.DynamicTable` with a string ``name`` and string
+``description``.
 
-###############################################################################
-# Constructing a table
-# --------------------
-# To create a :py:class:`~hdmf.common.table.DynamicTable`, call the constructor for
-# :py:class:`~hdmf.common.table.DynamicTable` with a string ``name`` and string
-# ``description``.
+sphinx_gallery_thumbnail_path = 'figures/gallery_thumbnail_dynamictable.png'
```

Comment:
A sphinx quirk -- this needs to be in its own `#` comment at the beginning of the script.